src/cmd/build.js

```diff
@@ -53,10 +54,10 @@ export async function getDefaultLocalizedName(
   let extensionName: string = manifestData.name;
 
   try {
-    messageData = JSON.parse(await fs.readFile(messageFile));
+    messageData = parseJson(await fs.readFile(messageFile));
   } catch (error) {
-    throw new UsageError(
-      `Error reading or parsing file ${messageFile}: ${error}`);
+    error.fileName = messageFile;
```

Comment:
Instead of setting the `error.fileName` property on the raised error, we should use the JSON file name as the second parameter of the `parseJSON` call (e.g. `parseJSON(content, fileName)`), by looking at the examples in the 'parse-json' README, it seems that this is how the file name is supposed to be officially associated to the raised exception (the same is needed for the other `parseJSON` calls in "src/util/manifest.js").